homeworks/homework5/task1.py

```diff
@@ -0,0 +1,91 @@
+"""
+Необходимо создать 3 класса и взаимосвязь между ними (Student, Teacher,
+Homework)
+Наследование в этой задаче использовать не нужно.
+Для работы с временем использовать модуль datetime
+1. Homework
+2. Student
+3. Teacher
+
+PEP8 соблюдать строго.
+Всем перечисленным выше атрибутам и методам классов сохранить названия.
+К названием остальных переменных, классов и тд. подходить ответственно -
+давать логичные подходящие имена.
+"""
+import datetime
+
+
+class Homework:
+    """Атрибуты:
+        text - текст задания
+        deadline - хранит объект datetime.timedelta с количеством
+        дней на выполнение
+        created - c точной датой и временем создания
+    Методы:
+        is_active - проверяет не истекло ли время на выполнение задания,
+        возвращает boolean
+    """
+    def __init__(self, text: str, deadline: datetime.timedelta,
+                 created=datetime.datetime.now()):
```

Comment:
you dont get this parameter, you should implement ith when you're creating object